assets/scss/components/_report-error.scss

```diff
@@ -0,0 +1,40 @@
+/*
+Ajax Partial Load
+
+Styles to contain a page partial loaded into the current page by AJAX, and a loading spinner to be displayed while the content is being loaded.
+This is used on the 'Get help with this page' content on the templates of many front-ends.
+
```

Comment:
Can you add a link to /assets/javascripts/modules/reportAProblem.js as this isn't really a component without it